vars/atomicHostTests.groovy

```diff
@@ -0,0 +1,107 @@
+import org.centos.Utils
+import org.centos.pipeline.PipelineUtils
+
+def call(body) {
+
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def getUtils = new Utils()
```

Comment:
I am not crazy about these var names...getXYZ...there is no "getting". better just to be **messageUtils** or **pipelineUtils**